adafruit_trellism4.py

```diff
@@ -82,8 +82,51 @@ def __setitem__(self, index, value):
             raise IndexError("Pixel assignment outside available coordinates.")
 
         self._neopixel[offset] = value
+
+    def show(self):
+        """
```

Comment:
Please update the docstrings to the following:
```
    def show(self):
        """
        Shows the new colors on the pixels themselves if they haven't already
        been autowritten.

        Use when ``auto_write`` is set to ``False``.

        The colors may or may not be showing after this function returns because
        it may be done asynchronously.
        """
        self._neopixel.show()

    @property
    def auto_write(self):
        """
        True if the neopixels should immediately change when set. If False,
        ``show`` must be called explicitly.

        This example disables ``auto_write``, sets every pixel, calls ``show()``, then
        re-enables ``auto-write``.

        .. code-block:: python
```

Reply:
Works for me. Fixed.